src/audio/aria/aria_hifi3.c

```diff
@@ -82,6 +83,8 @@ void aria_algo_get_data_odd_channel(struct processing_module *mod,
 	const int inc = sizeof(ae_int32);
 	ae_int32x2 gain;
 	const int ch_n = cd->chan_cnt;
+	const int shift_bits = 31 - cd->att;
```

Comment:
Does this mean that aria needs different "att" initialize for different format? It would be more efficient to have a different processing core for s24_4le and s32_le formats as for many other components.

Reply:
The concept of the fix has changed and this problem no longer exists.